### ISSUE TITLE
protonvpn does not change dns

### ISSUE THREAD

**Issue by bless-9** [corrad1nho/qomui#87]
Linux only changes ip and doesn't change dns. Windows using there official app dns is also included. not sure why Linux dns wouldn't be protected, unless I'm doing something wrong.

**Comment by corrad1nho** [comment]
Sorry for the late answer, I'll investigate this issue. Could you share the content of "etc/resolv.conf"?

**Comment by lambtho12** [comment]
I have the same issue. DNS is leaking with ProtonVPN. I tested it on dnsleaktest.com, and all the servers listed are from my ISP.

/etc/resolv.conf gives
```#modified by Qomui
nameserver 10.8.8.1
```
I also have plenty of time the following error in my system logs, not sure if it is related to Qomui
```[system] Would reject message, 3 matched rules; type="method_call", sender=":1.4" (uid=0 pid=427 comm="/usr/bin/NetworkManager --no-daemon ") interface="org.freedesktop.resolve1.Manager" member="ResolveHostname" error name="(unset)" requested_reply="0" destination="org.freedesktop.resolve1" (uid=980 pid=412 comm="/usr/lib/systemd/systemd-resolved ")```

**Comment by corrad1nho** [comment]
The DNS server in /etc/resolv.conf is correct. I suspect its systemd-resolved that's interfering. I could implement a check whether the systemd-resolved service is running or not and disable it while qomui is active. Could you check if there are no DNS leaks when you stop systemd-resolved?
`sudo systemctl stop systemd-resolved`

**Comment by lambtho12** [comment]
Indeed, stopping `systemd-resolved` seems to do the trick. I also had to disable it, otherwise it kept restarting on its own.
